### DISCUSSION TITLE
how to change cmdline cursor style

### DISCUSSION THREAD

**Discussion by dnxxymcj** [LazyVim/LazyVim#227] [Q&A]
I would like to change to beam .  🙏Thanks.

<img width="630" alt="1" src="https://user-images.githubusercontent.com/7660888/218099282-df949b07-956e-46ea-a5c6-6fc68bfd3a6e.png">

**Comment by leonasdev** [comment]
Add 
```lua
vim.opt.guicursor:append("c:ver1")
```
to your `~/.config/nvim/lua/config/options.lua`

**Reply by dnxxymcj** [reply]
thanks, but it's not work. 😭

**Reply by leonasdev** [reply]
> thanks, but it's not work. 😭

Oh, I see, that `folke/noice`'s issue.
